Mixpanel/MPNotification.m

```diff
@@ -1,9 +1,51 @@
 #import "MPLogger.h"
 #import "MPNotification.h"
 #import "Mixpanel.h"
+#import "MPNetworkPrivate.h"
 
+NSString* const PROPERTY_EVAL_FUNC_NAME = @"evaluateFilters";
+NSString* const PROPERTY_FILTERS_JS_URL = @"http://cdn-dev.mxpnl.com/libs/mixpanel.property-filters.dev.js";
 @implementation MPNotification
 
+#if !TARGET_OS_WATCH
++ (JSValue*) propertyFilterFunc {
+    static JSValue* propertyFilterFunc = nil;
```

Comment:
nit: `ClassName *variableName` instead of `ClassName* variableName` . Nothing wrong, just more consistent with existing coding style. The same comment will apply to all in this PR.